packages/block-library/src/latest-posts/edit.js

```diff
@@ -148,6 +149,22 @@ export default function LatestPostsEdit( { attributes, setAttributes } ) {
 		]
 	);
 
+	// If a user clicks to a link prevent redirection and show a warning.
+	const { createWarningNotice, removeNotice } = useDispatch( noticeStore );
+	let noticeId;
+	const showRedirectionPreventedNotice = ( event ) => {
+		event.preventDefault();
+		// Always remove previous warning if any to show one at a time
+		// and use a unique id to announce the message in screen readers
+		// in every click.
```

Comment:
@alexstine do you know if  we need to announce the warning in every click?

Reply:
@ntsekouras I think just once per block would be good enough. Is that too hard to do for tracking the state?

Reply:
done 😄